src/main/java/dev/ikm/tinkar/sandbox/TinkarStarterData.java

```diff
@@ -2735,6 +2751,15 @@ private static void configureConceptsAndPatterns(StarterData starterData, UUIDUt
                 .tinkarBaseModelMembership()
                 .build();
 
+        //converting above concept to composer api format.
+        session.compose((ConceptAssembler conceptAssembler) -> conceptAssembler.concept(TinkarTerm.ROOT_VERTEX))
```

Comment:
Do you know if this still generates the starter data correctly? My only concern is that I don't know if we can mix/match the Composer API with the older way. A quick test would be to run this to generate the starter data, then bring it up in Komet to make sure the concept navigation looks correct.

Reply:
Good luck with this fellows.